app/models/repository.rb

```diff
@@ -42,7 +43,22 @@ class Repository < ApplicationRecord
   has_many :exercises, dependent: :restrict_with_error
 
   def full_path
-    Pathname.new File.join(ACTIVITY_LOCATIONS, path)
+    if path.present?
+      Pathname.new File.join(ACTIVITY_LOCATIONS, path)
+    else
+      ''
+    end
```

Comment:
```suggestion
    Pathname.new File.join(ACTIVITY_LOCATIONS, path)
```
This is only something that occurs during testing, so we don't want to handle this in the model.